tests/resources/dspa-lite.yaml

```
@@ -8,7 +8,7 @@ spec:
   apiServer:
     deploy: true
     enableOauth: false
-    enableSamplePipeline: true
+    enableSamplePipeline: true    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)